string asset_id
---
geometry_msgs/Pose last_known_pose
